.travis.yml

```diff
@@ -32,6 +32,10 @@ before_install:
         conda install --yes scipy matplotlib nose coverage setuptools;
         travis_retry pip install -q coveralls;
       fi;
+    # install AVbin
+    - curl https://cloud.github.com/downloads/AVbin/AVbin/install-avbin-linux-x86-64-v10 -o avbin.sh
+    - chmod +x avbin.sh
+    - sudo bash -c "yes | ./avbin.sh"
```

Comment:
thanks.  FYI, using `curl` instead of `wget` had hiccups of its own: the public URL (`github.com/downloads/...`) redirected to the `cloud.github.com/...` address seen here, and `curl` was initially downloading only the redirect page, not its target (`wget` seems to have been clever enough to follow the redirect).  Noting here, as I don't know how stable the `cloud.github.com` address will be in future.